.changeset/large-penguins-yell.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kubernetes': patch
+'@backstage/plugin-kubernetes-backend': patch
```

Comment:
Don't need this package bumped here :)

Reply:
sure :)

Reply:
Sorry @samiramkr you'll still need to bump `'@backstage/plugin-kubernetes': patch` just not `kuberenetes-backend` :_